### ISSUE TITLE
Cant get ads server running

### ISSUE THREAD

**Issue by para2022** [WWBN/AVideo#8580]
I cant get ads server running. I get this error.

Cross-Origin Request Blocked: The Same Origin Policy disallows reading the remote resource at https://ourtube.co.uk/plugin/AD_Server/VMAP.php?video_length=149&vmap_id=l4jhf1q9n4vagrv711g754njk0&random=653a310fd40be&vmaps=W3sidGltZU9mZnNldCI6InN0YXJ0IiwidGltZU9mZnNldFNlY29uZHMiOjAsIlZBU1QiOnsiaWQiOjEsImNhbXBhaW5nIjo4MDB9LCJpZFRhZyI6InByZXJvbGwtYWQtMSJ9XQ%3D%3D. (Reason: Multiple CORS header ‘Access-Control-Allow-Origin’ not allowed).

**Comment by DanielnetoDotCom** [comment]
![image](https://github.com/WWBN/AVideo/assets/18009293/81493bd2-6fdf-4b76-b04e-28eb077d9f8a)

**Comment by para2022** [comment]
yes i turned it off as its not working so their was no pointing in having it turned on

**Comment by DanielnetoDotCom** [comment]
share the full js console logs
